examples/attention_is_all_you_need.py

```diff
@@ -197,17 +200,21 @@ def _create_mask(ops, lengths, max_sent):
     return mask
 
 
-def get_loss(ops, Yh, Y_docs):
+def get_loss(ops, Yh, Y_docs, Y_mask):
     Y_ids = docs2ids(Y_docs)
     nC = Yh.shape[-1]
     Y = [to_categorical(y, nb_classes=nC) for y in Y_ids]
     nL = max(Yh.shape[1], max(y.shape[0] for y in Y))
     Y, _, _ = ops.square_sequences(Y, pad_to=nL)
-    return Yh-Y.transpose((1, 0, 2))
+    Y = Y.transpose((1, 0, 2))
+    # Need the mask to get this right I think? Not sure what to do.
```

Comment:
No, I don't think so. If model is trained correctly, it is supposed to return <pad> to tokens that are not out of the mask scope.